mleap-tensorflow/src/main/scala/ml/combust/mleap/tensorflow/TensorflowModel.scala

```diff
@@ -21,12 +22,12 @@ case class TensorflowModel(@transient var graph: Option[tensorflow.Graph] = None
                            graphBytes: Array[Byte]) extends Model with AutoCloseable {
 
   def apply(values: Tensor[_] *): Seq[Any] = {
-    val garbage: mutable.ArrayBuilder[tensorflow.Tensor[_]] = mutable.ArrayBuilder.make[tensorflow.Tensor[_]]()
+    val garbage: mutable.ArrayBuilder[tensorflow.Tensor] = mutable.ArrayBuilder.make[tensorflow.Tensor]()
```

Comment:
Do you mean keep using the `dataType` in `MleapConverter.convert` method?
But we can get the dataType information form `tensor` object, use `schema` to cast it might cause trouble.

Reply:
@ancasarb Are you okay with just inferring the type from the tensor that is passed in?

Reply:
yeah, okay, let's see how this goes :) we can always add it back in i guess.

Reply:
Well `Casting.cast` is a good point.  I don’t have a very good solution for that. 
In the old code `dense.values.asInstanceOf[Array[Float]]`. I think we can not cast a buffer of type A to  a buffer B.
Maybe float32 --> int32 can, but not all it will cause segment fault. If we are going to support cast, we might needs to check source type and target type first, if they are the same, we copy the buffer, if they are not the same, well, it’s going to be slow, because we needs to assign one by one.
That’s the main reason I don’t want to do this in Tensor conversion, If they are not the same (which should be), I rather fail it